app/src/main/java/com/example/marvelapp/data/service/implementation/MarvelServiceFragImpl.kt

```diff
@@ -0,0 +1,28 @@
+package com.example.marvelapp.data.service.implementation
+
+import com.example.marvelapp.data.mapper.CharacterMapper.transformSingleCharacter
+import com.example.marvelapp.data.request.generator.MarvelRequestGenerator
+import com.example.marvelapp.data.service.api.MarvelApi
+import com.example.marvelapp.domain.entity.CharacterDetailEntity
+import com.example.marvelapp.domain.entity.CharacterEntity
+import com.example.marvelapp.domain.service.MarvelServiceCharacterDetail
+import io.reactivex.rxjava3.core.Observable
+
+class MarvelServiceFragImpl : MarvelServiceCharacterDetail {
```

Comment:
rename this class, because you not implementa a "Frag"

Reply:
Done thx